### DISCUSSION TITLE
not a vaild JSON? (SOLVED)

### DISCUSSION THREAD

**Discussion by Elonel5** [hlky/stable-diffusion-webui#918] [Q&A] [answered]
**Solution bellow**

My webui starts (having previously worked) and on loading loops the regular boot but ends with

> OSError: It looks like the config file at 'C:\Users\User/.cache\huggingface\transformers\9c24e6cd9f499d02c4f21a033736dabd365962dc80fe3aeb57a8f85ea45a20a3.26fead7ea4f0f843f6eb4055dfd25693f1a71f3c6871b184042d4b126244e142' is not a valid JSON file. 

(the folder has two files with the same name, one is a .json, and the other one is a 26FEAD7EA4F0F843F6EB4055DFD25693F1A71F3C6871B184042D4B126244E142 Fileb(???))

What can I do about it?

**Solution** 
A thread about it said to open the file (the one ending in 142 without the .json extension) and remove commas after "projection_dim": 768

I had a different text altogether, so I pasted this code inside (provided by [Anaphylaxis](https://github.com/Anaphylaxis) ) and it worked:

> { "_name_or_path": "clip-vit-large-patch14/", "architectures": [ "CLIPModel" ], "initializer_factor": 1.0, "logit_scale_init_value": 2.6592, "model_type": "clip", "projection_dim": 768, "text_config": { "_name_or_path": "", "add_cross_attention": false, "architectures": null, "attention_dropout": 0.0, "bad_words_ids": null, "bos_token_id": 0, "chunk_size_feed_forward": 0, "cross_attention_hidden_size": null, "decoder_start_token_id": null, "diversity_penalty": 0.0, "do_sample": false, "dropout": 0.0, "early_stopping": false, "encoder_no_repeat_ngram_size": 0, "eos_token_id": 2, "finetuning_task": null, "forced_bos_token_id": null, "forced_eos_token_id": null, "hidden_act": "quick_gelu", "hidden_size": 768, "id2label": { "0": "LABEL_0", "1": "LABEL_1" }, "initializer_factor": 1.0, "initializer_range": 0.02, "intermediate_size": 3072, "is_decoder": false, "is_encoder_decoder": false, "label2id": { "LABEL_0": 0, "LABEL_1": 1 }, "layer_norm_eps": 1e-05, "length_penalty": 1.0, "max_length": 20, "max_position_embeddings": 77, "min_length": 0, "model_type": "clip_text_model", "no_repeat_ngram_size": 0, "num_attention_heads": 12, "num_beam_groups": 1, "num_beams": 1, "num_hidden_layers": 12, "num_return_sequences": 1, "output_attentions": false, "output_hidden_states": false, "output_scores": false, "pad_token_id": 1, "prefix": null, "problem_type": null, "projection_dim" : 768, "pruned_heads": {}, "remove_invalid_values": false, "repetition_penalty": 1.0, "return_dict": true, "return_dict_in_generate": false, "sep_token_id": null, "task_specific_params": null, "temperature": 1.0, "tie_encoder_decoder": false, "tie_word_embeddings": true, "tokenizer_class": null, "top_k": 50, "top_p": 1.0, "torch_dtype": null, "torchscript": false, "transformers_version": "4.16.0.dev0", "use_bfloat16": false, "vocab_size": 49408 }, "text_config_dict": { "hidden_size": 768, "intermediate_size": 3072, "num_attention_heads": 12, "num_hidden_layers": 12, "projection_dim": 768 }, "torch_dtype": "float32", "transformers_version": null, "vision_config": { "_name_or_path": "", "add_cross_attention": false, "architectures": null, "attention_dropout": 0.0, "bad_words_ids": null, "bos_token_id": null, "chunk_size_feed_forward": 0, "cross_attention_hidden_size": null, "decoder_start_token_id": null, "diversity_penalty": 0.0, "do_sample": false, "dropout": 0.0, "early_stopping": false, "encoder_no_repeat_ngram_size": 0, "eos_token_id": null, "finetuning_task": null, "forced_bos_token_id": null, "forced_eos_token_id": null, "hidden_act": "quick_gelu", "hidden_size": 1024, "id2label": { "0": "LABEL_0", "1": "LABEL_1" }, "image_size": 224, "initializer_factor": 1.0, "initializer_range": 0.02, "intermediate_size": 4096, "is_decoder": false, "is_encoder_decoder": false, "label2id": { "LABEL_0": 0, "LABEL_1": 1 }, "layer_norm_eps": 1e-05, "length_penalty": 1.0, "max_length": 20, "min_length": 0, "model_type": "clip_vision_model", "no_repeat_ngram_size": 0, "num_attention_heads": 16, "num_beam_groups": 1, "num_beams": 1, "num_hidden_layers": 24, "num_return_sequences": 1, "output_attentions": false, "output_hidden_states": false, "output_scores": false, "pad_token_id": null, "patch_size": 14, "prefix": null, "problem_type": null, "projection_dim" : 768, "pruned_heads": {}, "remove_invalid_values": false, "repetition_penalty": 1.0, "return_dict": true, "return_dict_in_generate": false, "sep_token_id": null, "task_specific_params": null, "temperature": 1.0, "tie_encoder_decoder": false, "tie_word_embeddings": true, "tokenizer_class": null, "top_k": 50, "top_p": 1.0, "torch_dtype": null, "torchscript": false, "transformers_version": "4.16.0.dev0", "use_bfloat16": false }, "vision_config_dict": { "hidden_size": 1024, "intermediate_size": 4096, "num_attention_heads": 16, "num_hidden_layers": 24, "patch_size": 14, "projection_dim": 768 } }

**Comment by lolocler** [comment]
same error here. worked fine before reboot.


Relauncher: Launching...
Found GFPGAN
Found RealESRGAN
LDSR not found at path, please make sure you have cloned the LDSR repo to ./src/latent-diffusion/
Loading model from models/ldm/stable-diffusion-v1/model.ckpt
Global Step: 470000
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
making attention of type 'vanilla' with 512 in_channels
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
making attention of type 'vanilla' with 512 in_channels
Traceback (most recent call last):
  File "C:\Users\epong\.conda\envs\ldm\lib\site-packages\transformers\configuration_utils.py", line 650, in _get_config_dict
    config_dict = cls._dict_from_json_file(resolved_config_file)
  File "C:\Users\epong\.conda\envs\ldm\lib\site-packages\transformers\configuration_utils.py", line 734, in _dict_from_json_file
    return json.loads(text)
  File "C:\Users\epong\.conda\envs\ldm\lib\json\__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "C:\Users\epong\.conda\envs\ldm\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\epong\.conda\envs\ldm\lib\json\decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 88 column 3 (char 2317)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "scripts/webui.py", line 453, in <module>
    model, device,config = load_SD_model()
  File "scripts/webui.py", line 444, in load_SD_model
    model = load_model_from_config(config, opt.ckpt)
  File "scripts/webui.py", line 150, in load_model_from_config
    model = instantiate_from_config(config.model)
  File "c:\users\epong\stable-diffusion-webui\ldm\util.py", line 85, in instantiate_from_config
    return get_obj_from_str(config["target"])(**config.get("params", dict()))
  File "c:\users\epong\stable-diffusion-webui\ldm\models\diffusion\ddpm.py", line 461, in __init__
    self.instantiate_cond_stage(cond_stage_config)
  File "c:\users\epong\stable-diffusion-webui\ldm\models\diffusion\ddpm.py", line 519, in instantiate_cond_stage
    model = instantiate_from_config(config)
  File "c:\users\epong\stable-diffusion-webui\ldm\util.py", line 85, in instantiate_from_config
    return get_obj_from_str(config["target"])(**config.get("params", dict()))
  File "c:\users\epong\stable-diffusion-webui\ldm\modules\encoders\modules.py", line 142, in __init__
    self.transformer = CLIPTextModel.from_pretrained(version)
  File "C:\Users\epong\.conda\envs\ldm\lib\site-packages\transformers\modeling_utils.py", line 1764, in from_pretrained
    config, model_kwargs = cls.config_class.from_pretrained(
  File "C:\Users\epong\.conda\envs\ldm\lib\site-packages\transformers\models\clip\configuration_clip.py", line 126, in from_pretrained
    config_dict, kwargs = cls.get_config_dict(pretrained_model_name_or_path, **kwargs)
  File "C:\Users\epong\.conda\envs\ldm\lib\site-packages\transformers\configuration_utils.py", line 553, in get_config_dict
    config_dict, kwargs = cls._get_config_dict(pretrained_model_name_or_path, **kwargs)
  File "C:\Users\epong\.conda\envs\ldm\lib\site-packages\transformers\configuration_utils.py", line 652, in _get_config_dict
    raise EnvironmentError(
OSError: It looks like the config file at 'C:\Users\epong/.cache\huggingface\transformers\9c24e6cd9f499d02c4f21a033736dabd365962dc80fe3aeb57a8f85ea45a20a3.26fead7ea4f0f843f6eb4055dfd25693f1a71f3c6871b184042d4b126244e142' is not a valid JSON file.
Relauncher: Process is ending. Relaunching in 1s...
        Relaunch count: 1
Relauncher: Launching...

**Comment by codedealer** [comment, marked as answer]
#919 has a solution to this
